doc/introduction.md

```diff
@@ -35,7 +35,7 @@ For questions about how the project is organized, see the [project](./project_go
 ## Development
 
 * [Getting Started](./guides/getting_started/src/README.md)
-* [User Guides]({{< relref "doc/ug" >}})
+* [User Guides](https://docs.opentitan.org/doc/guides/getting_started/)
```

Comment:
These don't exist anymore in the new world. Most of this has moved to contributing.

Reply:
https://github.com/lowRISC/opentitan/pull/17323#discussion_r1121768123